CHANGES.rst

```diff
@@ -127,6 +127,9 @@ astropy.stats
 astropy.table
 ^^^^^^^^^^^^^
 
+- Accessing variable length columns from VO table no longer returns bytes
```

Comment:
This should be in the API changes section.  The more fundamental change is to the table item access behavior of a `Column` of `object` dtype whenever the object is `bytes`.  So in fact the VO table does still return a column of `bytes` objects, but it looks like `str` whenever they get accessed.

Reply:
Okay, I solidified the change log entry.